### ISSUE TITLE
Parsing multiple strings in HTML

### ISSUE THREAD

**Issue by julienmiclo** [intlify/vue-i18n-next#409]
Hello,
i read documentation and search in issues but i didn't find an answer for my question. It's possible to parse array of multiple strings before rendering like this :

```
  cgv: {
    contents: [
      {
        title: "Titre #1",
        paragraphs: [
            "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
            "In sollicitudin dui in placerat vehicula. Praesent ante mi, imperdiet sed lectus sed, pretium efficitur risus. Aenean at enim a enim consectetur consectetur vitae sit amet orci.",
            "Donec vulputate imperdiet neque sit amet imperdiet. Nulla in justo vitae diam tincidunt pellentesque vitae ut metus. Donec vel sapien a elit hendrerit lacinia eu a nibh. Etiam tempus mauris venenatis suscipit venenatis."
        ]
      },
{
        title: "Titre #2",
        paragraphs: [
            "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
            "In sollicitudin dui in placerat vehicula. Praesent ante mi, imperdiet sed lectus sed, pretium efficitur risus. Aenean at enim a enim consectetur consectetur vitae sit amet orci.",
        ]
      }
    ]
  }
```

result :
```
<h2>Titre #1</h2>
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
<p>In sollicitudin dui in placerat vehicula. Praesent ante mi, imperdiet sed lectus sed, pretium efficitur risus. Aenean at enim a enim consectetur consectetur vitae sit amet orci.</p>
<p>Donec vulputate imperdiet neque sit amet imperdiet. Nulla in justo vitae diam tincidunt pellentesque vitae ut metus. Donec vel sapien a elit hendrerit lacinia eu a nibh. Etiam tempus mauris venenatis suscipit venenatis.</p>
```

Thank you !

**Comment by kazupon** [comment]
Thank you for your feedback!

If you want to handle an array-like `cgv.contents`, you need to use `tm`.

<template v-for="content in tm('cvg.contents')">
  <h2>{{ content.title }}</h2>
  <p v-for="paragraph in content.paragraphs">
    {{ paragraph }}
  </p>
</template>

`tm` will return a reactive value. If you change the locale, `tm` will reactively return the value corresponding to the locale.

**Comment by kazupon** [comment]
However, I realized that this does not work when I pre-compile i18n resources. i18n resources pre-compile compiles all locale messages as message functions.

The message function needs to be run through the vue-i18n runtime. These message functions cannot be executed directly. It needs to be extending `t` or add a new API.

I will try to fix this issue.

**Comment by kazupon** [comment]
I've just fixed!
I've added new API `rt` (`$tr`).
This fix will be available after the release of v9.1.

**Comment by julienmiclo** [comment]
Merci !
